opennlp-tools/src/main/java/opennlp/tools/util/featuregen/GeneratorFactory.java

```diff
@@ -625,6 +625,38 @@ static void register(Map<String, XmlFeatureGeneratorFactory> factoryMap) {
     }
   }
 
+
+
+  /**
+   * @see TokenNormalizerFeatureGenerator
+   */
+  static class TokenNormalizerFeatureGeneratorFactory implements XmlFeatureGeneratorFactory {
+
+    public AdaptiveFeatureGenerator create(Element generatorElement,
+                                           FeatureGeneratorResourceProvider resourceManager)
+        throws InvalidFormatException {
+
+      Collection<AdaptiveFeatureGenerator> aggregatedGenerators = new LinkedList<>();
+
+      NodeList childNodes = generatorElement.getChildNodes();
+
+      for (int i = 0; i < childNodes.getLength(); i++) {
+        Node childNode = childNodes.item(i);
+        if (childNode instanceof Element) {
+          Element aggregatedGeneratorElement = (Element) childNode;
+          aggregatedGenerators.add(
+              GeneratorFactory.createGenerator(aggregatedGeneratorElement, resourceManager));
+        }
+      }
+      return new TokenNormalizerFeatureGenerator(new AggregatedFeatureGenerator(aggregatedGenerators.toArray(
```

Comment:
Question: is this line exceeding the 100 char limit? consider splitting to next line